src/vm/clrvarargs.cpp

```diff
@@ -33,10 +33,14 @@ void VARARGS::MarshalToManagedVaList(va_list va, VARARGS *dataout)
 {
     WRAPPER_NO_CONTRACT
 
+#if !(defined(_TARGET_ARM_) && defined(PLATFORM_UNIX))
```

Comment:
ARM has a different ABI to other platforms on Unix, as defined at http://infocenter.arm.com/help/topic/com.arm.doc.ihi0042e/IHI0042E_aapcs.pdf . However, having since finding this it seems we could handle the difference on ARM fairly easily.